app/src/test/java/com/github/swent/echo/viewmodels/HomeScreenViewModelTest.kt

```diff
@@ -203,4 +205,37 @@ class HomeScreenViewModelTest {
         homeScreenViewModel.onSortByChanged(v9)
         assertEquals(homeScreenViewModel.filtersContainer.value.sortBy, v9)
     }
+
+    @Test
+    fun refreshEventsTest() = runBlocking {
+        // Arrange
+        val oldEvents = homeScreenViewModel.displayEventList.value
+        val newEvents =
+            listOf(
+                Event(
+                    eventId = "newEvent",
+                    creator = EventCreator("a", ""),
+                    organizer = Association("a", "a", ""),
+                    title = "New Event",
+                    description = "",
+                    location = Location("Location 2", MAP_CENTER.toLatLng()),
+                    startDate = ZonedDateTime.now(),
+                    endDate = ZonedDateTime.now(),
+                    tags = setOf(Tag("1", "new")),
+                    participantCount = 5,
+                    maxParticipants = 8,
+                    imageId = 0
+                )
```

Comment:
I will do this next sprint, so it doesn't really matter that much, but it's preferable to use  `Event.EMTPY.copy(...)` instead of `Event(...)`. This goes for most other data classes as well.

That way there's no need to specify the fields that don't matter for the test, and it also improves the overall maintainability of the code (e.g. if changes are made to the `Event` data class there would be no need to refactor every single use of the `Event` constructor).